core/src/toga/widgets/scrollcontainer.py

```diff
@@ -6,6 +6,9 @@
 
 
 class ScrollContainer(Widget):
+    _MIN_WIDTH = 0
+    _MIN_HEIGHT = 0
```

Comment:
Not 100% convinced this is desirable. Box and Canvas both need to be able to collapse to 0 in case someone wants to use them for very small or 0 sized display elements - but ScrollContainer has to have scroll bars. Assuming a minimum size (historically 100x100, but I'm open to other suggestions) would seem like a sensible default, as otherwise you're potentially going to have a scroll container that can collapse to 0 size in a layout, and therefore be unusable.

Any user-specified size will always override this value - so if the use *explicitly* asks for a 10x10 scroll container, then they will get it - but a minimum value puts a floor on how small Pack will let it get by default.

Reply:
Agreed.